### PR TITLE
[usage] improve docs on playbooks

### PR DESCRIPTION
**Pull request by labo-flg**

### CODE REVIEW

File: docs/usage/automation.md [hunk 1]
```diff
@@ -135,10 +161,25 @@ You can also add all observables and relationships generated by this component i
...
+![Enroll entity in playbook](assets/playbook_enroll_entity.png)
+
+This will open a drawer where you can choose the playbook you want to trigger on this entity.
+
+![Enroll entity in playbook](assets/playbook_enroll_select.png)
+
+In this list, you will find:
+
+* active playbooks that have set an "Available for manual enrollment / trigger" event source
+* active playbooks with "Listen knowledge events" event source which filters match the entity
```
Comment (labo-flg) [inline docs/usage/automation.md:177]: I need a validation of this part.

The filters set in the "Available for manual enrollment / trigger" event source are not used ?

File: docs/usage/automation.md [hunk 2]
```diff
@@ -41,10 +41,35 @@ It is possible to create as many playbooks as needed which are running independe
 
 ![Create a new playbook](assets/playbook_create.png)
 
-The first step to define in the playbook is the “triggering event”, which can be any knowledge event (create, update or delete) with customizable filters. To do so, click on the grey rectangle in the center of the workspace and choose the component to "listen knowledge events". Configure it with adequate filters. You can use same filters as in other part of the platform.
+The first step to define in the playbook is your event source.
+To do so, click on the grey rectangle in the center of the workspace and select the right component.
```
Comment (labo-flg) [inline docs/usage/automation.md:45]:
```suggestion
To do so, click on the grey rectangle in the center of the workspace and select the input component that suits your needs.
```

File: docs/usage/automation.md [hunk 3]
```diff
@@ -41,10 +41,35 @@ It is possible to create as many playbooks as needed which are running independe
...
+
+![Choose input component](assets/playbook_input.png)
+
+### Listen knowledge events
+With this event source, the playbook will be triggered on any knowledge event (create, update or delete) with customizable filters.
```
Comment (labo-flg) [inline docs/usage/automation.md:50]:
```suggestion
With this event source, the playbook will be triggered on any knowledge event (create, update or delete) that matches the selected filters.
```

File: docs/usage/automation.md [hunk 4]
```diff
@@ -41,10 +41,35 @@ It is possible to create as many playbooks as needed which are running independe
...
+You can use same filters as in other part of the platform.
```
Comment (labo-flg) [inline docs/usage/automation.md:51]:
```suggestion
Note that you are limited to a subset of filters, available for stream events that contain STIX data objects.
```

**Review by nino-filigran — APPROVED** [review]
Looks good to me.

**Pull request merged**